### PR TITLE
Align Credentials for GSuiteAdmin, SymantecMSS, AlienVault_USM_Anywhere, AttackIQFireDrill

### PR DESCRIPTION
**Pull request by jlevypaloalto**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CIAC-7056](https://jira-hq.paloaltonetworks.local/browse/CIAC-7056)

## Description
Align Credentials for GSuiteAdmin, SymantecMSS, AlienVault_USM_Anywhere, AttackIQFireDrill.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by jlevypaloalto** [comment]
This PR must be force-merged as SymantecMSS has no unit-tests.

File: Packs/SymantecMSS/Integrations/SymantecMSS/SymantecMSS.py [hunk 1]
```diff
@@ -86,7 +87,7 @@ def api_call(body, headers):
         res = requests.post(url=SERVER_URL + "/SWS/incidents.asmx", cert=cert, data=body, headers=headers)
         if res.status_code < 200 or res.status_code >= 300:
             raise Exception(
-                "Got status code " + str(res.status_code) + " with body " + res.content + " with headers " + str(
+                "Got status code " + str(res.status_code) + " with body " + str(res.content) + " with headers " + str(
```
Comment (JudahSchwartz) [inline Packs/SymantecMSS/Integrations/SymantecMSS/SymantecMSS.py:90]: why this change?

**Review by JudahSchwartz — APPROVED** [review]

File: Packs/SymantecMSS/Integrations/SymantecMSS/SymantecMSS.py (hunk 1 above)
Comment (jlevypaloalto) [reply Packs/SymantecMSS/Integrations/SymantecMSS/SymantecMSS.py:90]: res.content can also be a bytes object so ruff or mypy were complaining.
